products/brms/_common/product.yml

```diff
@@ -6,22 +6,22 @@ documentation_minor_version: 6.0
 intro_video: https://vimeo.com/95600665
 vimeo_album: http://vimeo.com/album/2982444
 youtube_album: http://www.youtube.com/playlist?list=PLcrLvMVKO0b377pxbjvepIBuoQl7lcEiK
-guides: 
+guides:
   Business_Resource_Planner_Guide:
   Getting_Started_Guide:
   Release_Notes:
     formats:
-      html: 
+      html:
         url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_BRMS/6.0/html/6.0.2_Release_Notes/index.html
       html-single:
         url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_BRMS/6.0/html-single/6.0.2_Release_Notes/index.html
       pdf:
         url: https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_BRMS/6.0/pdf/6.0.2_Release_Notes/Red_Hat_JBoss_BRMS-6.0-6.0.2_Release_Notes-en-US.pdf
-      epub: 
+      epub:
         url: https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_BRMS/6.0/pdf/6.0.2_Release_Notes/Red_Hat_JBoss_BRMS-6.0-6.0.2_Release_Notes-en-US.pdf
   User_Guide:
 default_download_artifact_type: installer
-description: 'A rule-based programming platform built on Drools Expert, Drools Guvnor, and Drools Fusion that allows users to capture rules that can be easily modified as the business changes without impacting static applications.'
+description: 'A business process and decision management platform that combines Drools Expert, Drools Guvnor, and Drools Fusion in a single integrated distribution.'
```

Comment:
This is the wrong place to make this change. Do it here: https://github.com/jboss-developer/www.jboss.org/blob/master/products/index.adoc#brms as we want the text to change on http://www.jboss.org/products/